project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject toucan "1.1.2"
+(defproject toucan "1.1.4-SNAPSHOT"
```

Comment:
what happened to 1.1.3?

Reply:
It's on https://github.com/jesims/toucan/tree/refresh-hydrations (PR pending)